doc/docs/modules/ROOT/pages/procedures.adoc

```diff
@@ -113,6 +113,23 @@ Input Parameters:
 
 |===
 
+Configuration parameters:
+[cols="3*",options="header"]
+|===
+|Name
+|Type
+|Description
+
+|`key`
+|Object
+|The key of message that you want to stream
```

Comment:
actual behavior is that if the key specified doesn't exist, you get a random UUID and no warning, so you should note this right here.   Is this the correct behavior?  Note that if someone has a tiny typo in what they provide to the config, it might cause problems.  Imagine you have records with `customerId` and you say that `customerid` is the key.   Should this fail, or produce a random UUID which could cause problems downstream?

Reply:
I updated the documentation with the random UUID note. 
Moreover, I have modified "The key of message" with the most appropriate "The **value key** of message".
Your observation makes me think about a potential bug with  null key, so I've updated the code for this use case. 
Now if I have a query like `"CREATE (c:Customer {id :1}) WITH c CALL streams.publish('customers', c, {key: c.foo})"` the message is also sent with null key, and you use the partitioning system defined in the broker.